### ISSUE TITLE
Track searches that yield no results

### ISSUE THREAD

**Issue by toolness** [18F/calc#928]
At a meeting today Kelly mentioned that it'd be very useful to know what kind of searches users are doing that yield no results. This will help them figure out what schedules to add to CALC.

This should be pretty easy to do with GA!

(Related to #839.)

**Comment by jseppi** [comment]
Could this potentially be broadened to track result counts of all searches?

**Comment by toolness** [comment]
Yep definitely! I just wanted to capture the "user story" as accurately as possible, but at the implementation level we can definitely track general result counts, and maybe even additional metrics like the returned average price, std deviation etc.
